modules/ting_das/includes/ting_das.admin.inc

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file
+ * Define the administration interface for the module.
+ */
+
+/**
+ * Admin settings form for Ting DAS integration.
+ */
+function ting_das_admin_settings_form() {
+  $form = array();
+
+  $form['ting_das'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Digital article server'),
+    '#tree' => TRUE,
+  );
+
+  $defaults = variable_get('ting_das', array());
+  $form['ting_das']['url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('End point'),
+    '#description' => t('The service endpoint for "placeCopyRequest", e.g. https://developer.statsbiblioteket.dk/elba-webservices/services/placeCopyRequest'),
+    '#default_value' => isset($defaults['url']) ? $defaults['url'] : '',
+  );
```

Comment:
Idea: Drop the issets for the default value and place default values in the default array if `ting_das` is not set.

Example:

```php
$defaults = variable_get('ting_das', array(
  'url' => '')
);
$form['ting_das']['url'] = array(
    '#type' => 'textfield',
    '#title' => t('End point'),
    '#description' => t('The service endpoint for "placeCopyRequest", e.g. https://developer.statsbiblioteket.dk/elba-webservices/services/placeCopyRequest'),
    '#default_value' => $defaults['url'],
);

Reply:
Also: Should this - and other values be `#required`. The module will not work with a partial set configuration.

Reply:
Required added and isset removed and defaults moved to... well defaults array